_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,289 @@
+
+### Removing Apache 2.0 dependencies
+
+As we move towards a v3.0 release of MDAnalysis, we will be looking to replace existing [Apache 2.0][]
+core dependencies. Of particular impact, this may mean that access to reading [MMTF][] files may no longer
+be available by default through the MDAnalysis library, but will instead be offered through an optionally
+installable [MDAKit](https://mdakits.mdanalysis.org/about.html).
+
+## Getting in touch
+
+We understand that license changes are a complex subject that can have a large impact on how users and
+developers may interact with the MDAnalysis library. Should you have any concerns, questions, or comments,
+we urge you to get in touch with the core developer team. You can do so over our
+[public forums[({% link index.md %}#participating) or privately by emailing *licensing@mdanalysis.org*.
+
+## Definitions
+
+### package license
+
+The license under which the MDAnalysis library is packaged and released. This may be different from the
+but still compatible with the [source code license](#source-code-license).
+
+### source code license
+
+The license under which the MDAnalysis source code is released. This only accounts for the code itself
+and not any impact in licensing incurred from linking (i.e. `python importing`) to other libraries.
+
+### license compatibility
+
+Whether or not the terms of two licenses allows them to be compatible with each other.
+
+### copyleft
+
+Whether the terms of a given license enforces that other codes (even under other licenses) must be bound
+by the same conditions. See the [FSF definition](https://www.gnu.org/licenses/copyleft.en.html) for more information.
+
+## Acknowledgements
+
+We are very grateful for the administrative and legal support from our fiscal sponsor, [NumFOCUS][].
+
+This relicensing effort has been made possible in part by the CZI grant DAF2021-237663,
+grant DOI https://doi.org/10.37921/426590wiobus from the Chan Zuckerberg Initiative DAF,
+an advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989).
+
+– The MDAnalysis [Core Developers][]
+
+[OSI]: https://opensource.org/osd
+[tldr legal]: https://tldrlegal.com/
+[SSI licensing]: https://www.software.ac.uk/resources/guides/choosing-open-source-licence
+[OSD]: https://opensource.org/osd
+[GPL v2]: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+[GPL v3]: https://www.gnu.org/licenses/gpl-3.0.en.html
+[LGPL v2.1]: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+[LGPL v3]: https://www.gnu.org/licenses/lgpl-3.0.en.html
+[MIT]: https://opensource.org/license/mit/
+[BSD 3.0]: https://opensource.org/license/bsd-3-clause/
+[Apache 2.0]: https://www.apache.org/licenses/LICENSE-2.0
+[contributors]: https://github.com/MDAnalysis/mdanalysis/blob/develop/package/AUTHORS
+[NumFOCUS]: https://www.numfocus.org
+[expert opinions]: https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=heather+meeker+open+source+for+business&btnG=
```

Comment:
would it maybe be better to cite direct rather than use a google scholar link? (especially since it seems google scholar doesn't actually link you to anywhere to read this?)

Reply:
Couldn't really work this one out, so I just referenced the book in the text directly.

Reply:
Couldn't really work this one out, so I just referenced the book in the text directly.